src/dialog-controller.js

```diff
@@ -8,8 +8,10 @@ import {invokeLifecycle} from './lifecycle';
 export class DialogController {
   settings: any;
   constructor(renderer: DialogRenderer, settings: any, resolve: Function, reject: Function) {
+    let defaultSettings = renderer ? renderer.defaultSettings || {} : {};
```

Comment:
This was related to https://github.com/aurelia/dialog/issues/157. I wasn't sure but it looked as though people were trying to manually inject the DialogController using a container that didn't have a DialogRenderer attached to it. I wasn't entirely sure about the use case, but this apparently worked before the renderer refactor (if I remember correctly, the controller didn't need a reference to the renderer, so the renderer wasn't a constructor argument). If this is a questionable change, I'm okay with removing it.